docker-compose/compute_wrapper/Dockerfile

```diff
@@ -5,13 +5,13 @@ ARG TAG=latest
 FROM $REPOSITORY/${COMPUTE_IMAGE}:$TAG
 
 USER root
+#Faker is required for the pg_anon test
 RUN apt-get update &&       \
     apt-get install -y curl \
                        jq   \
                        python3-pip \
-                       netcat
-#Faker is required for the pg_anon test
-RUN pip3 install Faker
+                       netcat-traditional \
+                       faker
```

Comment:
I wouldn't do any unrelated changes to debian 12

Reply:
this is required for debian 12, otherwise will have to pass `--break-system-package` which look even worse 

```
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.

    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.

    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.

    See /usr/share/doc/python3.11/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```